app/templates/home.html

```diff
@@ -1,21 +1,34 @@
 {% extends "partials/index/index_base.html" %}
 
 {% block content %}
+<script src="https://unpkg.com/sweetalert/dist/sweetalert.min.js"></script>
+<script src="{{ url_for('static', path='/js/dates_calculator.js') }}"></script>
 
+<div class="container mt-4">
+</div>
 
-  <div class="container mt-4">
-  </div>
 
-
-  <!-- I chose to add the quote day to the home page until the relavent calendar view will be developed.  -->
-  <div>
+<!-- I chose to add the quote day to the home page until the relavent calendar view will be developed.  -->
+<div>
     {% if quote %}
-      {% if not quote.author %}
-        <p><i>"{{ quote.text }}"</i></p>
-      {% else %}
-        <p><i>"{{ quote.text }}"</i> <span style="font-size: small;">&nbsp; \ {{ quote.author }}</span></p>
-      {% endif %}
+    {% if not quote.author %}
+    <p><i>"{{ quote.text }}"</i></p>
+    {% else %}
+    <p><i>"{{ quote.text }}"</i> <span style="font-size: small;">&nbsp; \ {{ quote.author }}</span>
+    </p>
     {% endif %}
-  </div>
-
+    {% endif %}
+</div>
+<div class="form-style">
+    <h3>Time calculator:</h3>
+    <form>
+        <label for="startDate">Start date:</label>
+        <input id="startDate" name="date1" type="date">
+        <br>
+        <label for="endDate">End date:<em>&#x2a;</em></label>
+        <input id="endDate" name="date2" type="date">
+    </form>
+    <button type="button" onclick="hiddenDifference()">Calculate</button>
+    <h3 id="demo"></h3>
```

Comment:
What is the purpose of the `h3`? Should it be a title?

If not - prefer using `p`/`span`/`div` tags + CSS

Reply:
fixed